src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-bar"
+      data-qa="big"
+      action="#"
+      method="get"
+    >
+      <img
+        class="search-bar__loupe"
+        alt="search-loupe"
+        src="./images/Search.svg"
+      >
+      <input
+        class="search-bar__input"
+        placeholder="Try &quot;Los Angeles&quot;"
+        type="text"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form
+      class="search-bar"
+      data-qa="small"
+      action="#"
+      method="get"
+    >
+      <div>
```

Comment:
The div element wrapping the second search input field and its associated image is not necessary if there is no specific styling or functionality attached to it. Consider removing it if it's not needed.